lib/wiki.ml

```diff
@@ -325,15 +325,12 @@ let summary_of_tlsw_lines conf short lines =
     let lines =
       "<dl><dd>" :: "<table id=\"summary\" cellpadding=\"10\">" ::
       ("<tr><td align=\"" ^ conf.left ^ "\">") ::
-      ("<div style=\"text-align:center\"><b>" ^
-       capitale (message_txt conf 3) ^ "</b>") ::
-      "<script type=\"text/javascript\">" :: "//<![CDATA[" ::
-      "showTocToggle()" :: "//]]>" :: "</script>" :: "</div>" ::
-      "<div class=\"summary\" id=\"tocinside\">" ::
+      ("<a type=\"button\" class=\"btn btn-info\" \
```

Comment:
`type="button"` does not seem valid to me. Is it ? I guess that `role` would be fine.

Also, have you seen https://github.com/geneweb/geneweb/issues/778#issuecomment-515582386 which would be a solution keeping the cookie support ?